changelog/issue-6668.md

```diff
@@ -0,0 +1,5 @@
+level: patch
+audience: developers
+reference: issue 6668
+
```

Comment:
```suggestion
---
```

I think there's a separator missing, so one check fails

Reply:
Resolved: added the separator.